gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -421,6 +421,35 @@ TEST SUITE: None
 The `REVISION` value increments the `ingress` release by `1` each time you run this command with the same namespace, repository, and chart names. For example, if you run the command a second time, you find a new revision number (`REVISION: 2`). The `LAST DEPLOYED` date reflects the date of that revision.
 //    do not add an empty new line here at end, please
 // end::k8s-flex-helm-chart-deploy[]
```

Comment:
below adds local section that removes `--set gateway.mode=connected`; default is local, so `gateway.mode` is not needed in the local section